questions/323-prop-validation/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { mount } from "@vue/test-utils"
+import { describe, it, expect } from "vitest"
+
+import Button from "./Button.vue"
+
+describe("prop-validation", () => {
+  it("should have a default type prop of 'default'", () => {
+    const wrapper = mount(Button)
+    console.log(wrapper.vm)
```

Comment:
Let's remove the console output, thanks.

Reply:
got it